store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/RxSingleStoreTest.kt

```diff
@@ -50,7 +50,7 @@ class RxSingleStoreTest {
                     Completable.complete()
                 }
             )
-            .withScheduler(Schedulers.io())
+            .withScheduler(Schedulers.single())
```

Comment:
Since this is a test case, you can also pass `Schedulers.trampoline()`.

Reply:
Now this is getting more interesting. After changing it to `trampoline()`, the test fails consistently:

```
java.lang.AssertionError: Values at position 1 differ; expected: Data(value=3 2, origin=Fetcher) (class: Data) but was: Data(value=3 2, origin=Persister) (class: Data) (latch = 1, values = 2, errors = 0, completions = 0)
Expected :Data(value=3 2, origin=Fetcher) (class: Data) 
Actual   :Data(value=3 2, origin=Persister) (class: Data) (latch = 1, values = 2, errors = 0, completions = 0)
```

Any idea?